extensions/amp-auto-ads/0.1/attributes.js

```diff
@@ -27,6 +27,7 @@ const TAG = 'amp-auto-ads';
  */
 const NON_DATA_ATTRIBUTE_WHITELIST = {
   'type': true,
+  'rtc-config': true,
```

Comment:
I think it should be fine. but just in case I'm missing anything  @keithwrightbos